docs/concepts/store/README.md

```diff
@@ -114,6 +114,32 @@ export class ZooComponent {
 }
 ```
 
+#### `dispatch` Utility
+
+NGXS offers a utility function named `dispatch`, which takes an action as a parameter and returns a function. This function can then be called with parameters for the action constructor. Subsequently, the provided action is dispatched at the end:
+
+```ts
+import { dispatch } from '@ngxs/store';
+
+export class InvoiceLinesComponent {
+  getInvoiceLines = dispatch(GetInvoiceLines);
+
+  constructor() {
+    const route = inject(ActivatedRoute);
+
+    route.params
+      .pipe(
+        map(params => params.invoiceId),
+        distinctUntilChanged(),
+        takeUntilDestroyed()
+      )
+      .subscribe(invoiceId => {
+        this.getInvoiceLines(invoiceId);
+      });
+  }
```

Comment:
A simpler example would be better, so the utility is not lost in all the code.
Potentially:
```
// An action declared somewhere in your app
class Greet {
  static readonly type = 'Greet';
  constructor ( public greeting: string ) { }
}

// Then, in your component
export class MyComponent {
  greet = dispatch(Greet);

  constructor() {
    this.greet("Hello world!");
  }
}
```
It is good to mention the utility here, but I think it would be good to eventually have a page dedicated to these utilities.